pom.xml

```diff
@@ -31,26 +31,22 @@
 
   <modules>
     <module>api</module>
-    <module>api-1.11</module>
-    <module>api-2.0</module>
-    <module>omod-1.10</module>
-    <module>omod-2.0</module>
     <module>omod</module>
   </modules>
 
   <properties>
-    <openMRSVersion>1.10.5</openMRSVersion>
-    <openMRS2_0Version>2.0.0</openMRS2_0Version>
+    <openMRSVersion>2.4.0</openMRSVersion>
```

Comment:
Do we need this to be 2.4.0? Currently the target for OMRS 3 is 2.1+

Reply:
Ohh thanks @ibacher, the motivation for this refactor is to have the module usable with OMRS 3 and it makes sense indeed to begin depending on the minimum target openmrs platform version. However I see the minimum version currently pointing to [2.3.6-SNAPSHOT](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/49490f085fa7b0be5121bbf89dd09a2500bb53ce/package/pom.xml#L26), or could I be missing something @ibacher?

Reply:
Hi @ibacher, I'd think 2.2+ is fair enough to start with so that adding compatibility for platform versions below 2.2 may came when the need arises through tickets. This is because adding in compatibility for those versions extends the scope of this work covered here, i.e. introducing compatibility submodules.